src/components/Form/index.cy.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import ReactHookForm from 'react-hook-form';
+
+import { alertTestIds } from 'components/Alert';
+import { warningIconTestId } from 'components/Icon/Warning';
+import Input, { inputTestIds } from 'components/Input';
+
+import Form, { formTestIds } from '.';
+
+describe('Form', () => {
+  it('renders form children', () => {
+    cy.mount(
+      <Form>
+        <Input type="text" name="username" required />
+      </Form>
+    );
+
+    cy.findByTestId(formTestIds.form).should('be.visible');
+
+    cy.findByTestId(formTestIds.form).within(() => {
+      cy.findByTestId(inputTestIds.input).should('be.visible');
+    });
+  });
+
+  it('given there are no form errors, does NOT render form error', () => {
```

Comment:
Let's refactor the test description in this PR to put the context inside describe block

```
describe('given there are no form errors', () => {
  it(' ... ', () => {})
})
```

Reply:
Updated 👍 - [3d02646](https://github.com/rosle/react-nimble-survey/pull/25/commits/3d0264612ae6a92be1a08fb786b316156eb2adf1)